src/connectors/menu/connectMenu.js

```diff
@@ -139,6 +139,23 @@ export default function connectMenu(renderFn) {
         return this.isShowingMore ? showMoreLimit : limit;
       },
 
+      refine({ helper, items }) {
+        return facetValue => {
```

Comment:
When working with @marielaures we found that actually you don't need to use items to figure out the refined value. You can get it using `getHierarchicalFacetBreadcrumb` ([doc](https://community.algolia.com/algoliasearch-helper-js/reference.html#AlgoliaSearchHelper#getHierarchicalFacetBreadcrumb)), the first item will be the one currently selected if any.